python/ray/_private/worker.py

```diff
@@ -3268,7 +3268,7 @@ def method(self):
             By default it is empty.
         accelerator_type: If specified, requires that the task or actor run
             on a node with the specified type of accelerator.
-            See `ray.util.accelerators` for accelerator types.
+            See `ray.util.accelerators <https://github.com/ray-project/ray/blob/master/python/ray/util/accelerators/accelerators.py>`_ for accelerator types.
```

Comment:
You can add the # noqa: E501 at the ''' at the end of this whole block, similar to https://github.com/ray-project/ray/blob/master/ci/ray_ci/tester.py#L26

Reply:
TY!